src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,18 @@
+package core.basesyntax;
+
+public class Main {
+    public static void main(String[] args) {
+        FigureSupplier figureSupplier = new FigureSupplier();
+        final int figureArraySize = 6;
+        Figure[] figureArray = new Figure[figureArraySize];
+
+        for (int i = 0; i < figureArraySize; i++) {
+            if (i < figureArraySize / 2) {
+                figureArray[i] = figureSupplier.getRandomFigure();
+            } else {
+                figureArray[i] = figureSupplier.getDefaultFigure();
+            }
+            figureArray[i].draw();
+        }
```

Comment:
why do we need array? our task is just to create figures, we dont need to use them in future to store them in array for that
```suggestion
        for (int i = 0; i < figureArraySize; i++) {
            
            if (i < figureArraySize / 2) {
                figureSupplier.getRandomFigure().draw();
            } else {
                figureSupplier.getDefaultFigure().draw();
            }
        }
```